### PR TITLE
feat: add listings page and confirmation page

### PR DESCRIPTION
**Pull request by lbarad**
issue links: #66 #68

### CODE REVIEW

File: client/leasing-portal/src/App.tsx [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import "./App.css";
 import Header from "./components/Header";
 import Hero from "./components/Hero";
-import Filters from "./components/Filters";
 import Listings from "./components/Listings";
 import Properties from "./components/Properties";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
 import Footer from "./components/Footer";
+import './App.css';
```
Comment (rahulvemula) [inline client/leasing-portal/src/App.tsx:8]: Duplicate import. please remove

File: client/leasing-portal/src/App.tsx [hunk 2]
```diff
@@ -33,7 +35,9 @@ function App() {
             }
           />
           {/* TO-DO: add route to support page  */}
-          <Route path="/support" element={<Filters />} />
+          <Route path="/support" element={<ApptListing />} />
```
Comment (rahulvemula) [inline client/leasing-portal/src/App.tsx:38]: Needs to be changed as per mentioned To-Do

File: client/leasing-portal/src/App.tsx (hunk 1 above)
Comment (lbarad) [reply client/leasing-portal/src/App.tsx:8]: my bad, will remove!

File: client/leasing-portal/src/App.tsx (hunk 2 above)
Comment (lbarad) [reply client/leasing-portal/src/App.tsx:38]: yeah, will be handled in next sprint

**Review by rahulvemula — APPROVED** [review]
Looks good

**Pull request merged**